foundation/foundation-configuration-store/src/test/java/com/boozallen/aissemble/configuration/store/LoadConfigurationsSteps.java

```diff
@@ -48,43 +64,68 @@ public void cleanup() {
         response = null;
     }
 
-    @Given("URIs pointing to valid base and environment configurations")
-    public void URIsPointingToValidBaseAndEnvironmentConfigurations() {
+    @Given("a base URI indicating a directory housing valid base configurations")
+    public void URIsPointingToValidBaseConfigurations() {
         baseURI = "src/test/resources/configurations/base";
+    }
+
+    @And("an environment-specific URI indicating a directory housing valid environment-specific configurations")
+    public void URIsPointingToValidEnvironmentConfigurations() {
         environmentURI = "src/test/resources/configurations/example-env";
     }
+    
+    @When("the configuration service starts")
+    public void theConfigurationServiceStarts() {
+        configStoreInitMock.init();
```

Comment:
S: we don't need to run this init() separately, it's already being initialized when the web server starts

Reply:
hmm, we still need to call ConfigStoreInitTestHelper.init(), because the initialization isn't linked to a filestore (i.e. testPropertyDao in unit test), and the write function will be essentially void with default initialization

Reply:
It should be since we specified to use the TestPropertyDao class in the application.properties file.

Reply:
gotcha! but how do we retrieve the same  TestPropertyDao in the test? When we want to validate anything, we need to access to TestPropertyDao object initiated in the ConfigStoreInit during initialization process. However, we don't have any getter function in ConfigStoreInit

Reply:
But directly making a request to server, we can access to properties uploaded. I've got those fixed and removed the init() call